test/polygon.jl

```diff
@@ -11,6 +11,12 @@
     # valid country code
     polygon = Geography.get("IND")
     @test typeof(polygon) === ArchGDAL.IGeometry
+
+    level1 = Geography.get("IND", "Uttar Pradesh")
+    @test typeof(polygon) === ArchGDAL.IGeometry
```

Comment:
Nice. Now that we are querying more polygons, it would be nice to actually add more meaningful tests about the number of points encountered, and properties of the polygon loaded. We will need to know the API of IGeometry in order to convert it to a Meshes.jl/Polygon or GeometrySet.